crt_portal/static/sass/custom/form.scss

```diff
@@ -424,3 +424,10 @@ select.usa-input--error {
     }
   }
 }
+
+//Since #page-errors is in a parent report_base.html, and we don't want it in the pro form, we reference it via the #id_pro_form_view-current_step sibling.
+#id_pro_form_view-current_step ~ .crt-portal-card {
+  #page-errors {
+    display: none;
+  }
+}
```

Comment:
I just wanted to note that this has a potential to be brittle. It's highly dependent on two unrelated elements being siblings, and there's no guarantee that the layout remains structured in exactly the same way in the future. I would say that my preferred way of handling this is to have a class or id at a higher level in the template (e.g. the body tag, or the main element) that indicates what view this is, and that way `#page-errors` in any hierarchy of the DOM will still get targeted properly. I don't think it's worth fixing at this late in the game since it works as-is but it could help to note that this is prime for refactoring at some point down the line.